Doberman/BaseDevice.py

```diff
@@ -256,15 +255,17 @@ class LANDevice(Device):
     """
     Class for LAN-connected devices
     """
+    msg_wait = 1.0 # Seconds to wait for response
+    recv_interval = 0.1 # Socket polling interval
+    eol = b'\r'
```

Comment:
These should be class attributes, right? self.* in setup(self)

Reply:
If they go in setup they will be instance attributes, won't they? But all are constants, they can be declared once here like this avoided the setup (I think -- and it certainly works now ;) )

Reply:
I was just wondering because you use self.* in line 296ff

Reply:
You have reached the limit of my python expertise (or at least knowledge of the best way to write it)!
My understanding is that declaring it there makes it a class variable (which is probably correct here), and you can then access it (at least to read it) with either self.x or LANDevice.x

If you reassign it to a different value then it turns into an instance variable for that instance, but then you probably wanted the self.x anyway? It's actually very confusing. Maybe python isn't so intuitive after all :D

Reply:
No, you're absolutely right. I don't know why this confused me. Probably because I'm used to have constants written in UPPERCASE.